dictionaries/CallMap.php

```diff
@@ -7158,63 +7158,63 @@
 'maxdb_thread_safe' => ['bool'],
 'maxdb_use_result' => ['resource', 'link'=>''],
 'maxdb_warning_count' => ['int', 'link'=>'resource'],
-'mb_check_encoding' => ['bool', 'var='=>'string', 'encoding='=>'string'],
-'mb_chr' => ['string|false', 'cp'=>'int', 'encoding='=>'string'],
-'mb_convert_case' => ['string|false', 'sourcestring'=>'string', 'mode'=>'int', 'encoding='=>'string'],
-'mb_convert_encoding' => ['string', 'val'=>'string', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
-'mb_convert_encoding\'1' => ['array<int, string>', 'val'=>'array<int, string>', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
-'mb_convert_kana' => ['string|false', 'string'=>'string', 'option='=>'string', 'encoding='=>'string'],
-'mb_convert_variables' => ['string|false', 'to_encoding'=>'string', 'from_encoding'=>'array|string', '&rw_vars'=>'string|array|object', '&...rw_vars='=>'string|array|object'],
+'mb_check_encoding' => ['bool', 'value='=>'string', 'encoding='=>'string'],
+'mb_chr' => ['string|false', 'codepoint'=>'int', 'encoding='=>'string'],
+'mb_convert_case' => ['string|false', 'string'=>'string', 'mode'=>'int', 'encoding='=>'string'],
+'mb_convert_encoding' => ['string', 'string'=>'string', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
+'mb_convert_encoding\'1' => ['array<int, string>', 'string'=>'array<int, string>', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
+'mb_convert_kana' => ['string|false', 'string'=>'string', 'mode='=>'string', 'encoding='=>'string'],
+'mb_convert_variables' => ['string|false', 'to_encoding'=>'string', 'from_encoding'=>'array|string', '&rw_var'=>'string|array|object', '&...vars='=>'string|array|object'],
```

Comment:
I later discovered that scanf, fscanf, and mb_convert_variables would have `w_` stripped off by my script.

I think Psalm completely ignores the `(r|w|rw)_` prefix - I don't remember if it even strips it out anywhere; it probably uses plugins/stubs for reference logic

Reply:
It uses `&` for passed-by-reference vars – @orklah anyway you could update to preserve that character?

Reply:
Done. I also fixed an issue in mysqli bind_result: https://github.com/vimeo/psalm/pull/4745/commits/f508a5a34bee553b6507996376bf622e2389307b#diff-b8eaee1f550652657daf0a771be5e785bdb01e91acd004b4bbb4c41def706713L8460 . cc @TysonAndre